### ISSUE TITLE
[TE9][TC-IDM-1.2]Not meeting the expected results

### ISSUE THREAD

**Issue by nansenLV** [project-chip/connectedhomeip#18379]
[TC-IDM-1.2]step4 step5 step8 step9  doesn't seem to meet the expected results.

!!!EXCEPTED BEHAVIOR:

step4:

[1650608823029] [89418:1400921] CHIP: [DMG] InvokeResponseMessage =
[1650608823029] [89418:1400921] CHIP: [DMG] {
[1650608823029] [89418:1400921] CHIP: [DMG]         suppressResponse = false, 
[1650608823029] [89418:1400921] CHIP: [DMG]         InvokeResponseIBs =
[1650608823029] [89418:1400921] CHIP: [DMG]         [
[1650608823029] [89418:1400921] CHIP: [DMG]                 InvokeResponseIB =
[1650608823029] [89418:1400921] CHIP: [DMG]                 {
[1650608823029] [89418:1400921] CHIP: [DMG]                         CommandStatusIB =
[1650608823029] [89418:1400921] CHIP: [DMG]                         {
[1650608823029] [89418:1400921] CHIP: [DMG]                                 CommandPathIB =
[1650608823029] [89418:1400921] CHIP: [DMG]                                 {
[1650608823029] [89418:1400921] CHIP: [DMG]                                         EndpointId = 0x1,
[1650608823029] [89418:1400921] CHIP: [DMG]                                         ClusterId = 0x6,
[1650608823029] [89418:1400921] CHIP: [DMG]                                         CommandId = 0x1,
[1650608823029] [89418:1400921] CHIP: [DMG]                                 },
[1650608823029] [89418:1400921] CHIP: [DMG]                                 
[1650608823029] [89418:1400921] CHIP: [DMG]                                 StatusIB =
[1650608823029] [89418:1400921] CHIP: [DMG]                                 {
[1650608823029] [89418:1400921] CHIP: [DMG]                                         status = 0x7e (UNSUPPORTED_ACCESS),
[1650608823029] [89418:1400921] CHIP: [DMG]                                 },
[1650608823029] [89418:1400921] CHIP: [DMG]                                 
[1650608823029] [89418:1400921] CHIP: [DMG]                         },
[1650608823029] [89418:1400921] CHIP: [DMG]                         
[1650608823029] [89418:1400921] CHIP: [DMG]                 },
[1650608823029] [89418:1400921] CHIP: [DMG]                 
[1650608823029] [89418:1400921] CHIP: [DMG]         ],
[1650608823029] [89418:1400921] CHIP: [DMG]         
[1650608823029] [89418:1400921] CHIP: [DMG]         InteractionModelRevision = 1
[1650608823029] [89418:1400921] CHIP: [DMG] },
[1650608823029] [89418:1400921] CHIP: [DMG] Received Command Response Status for Endpoint=1 Cluster=0x0000_0006 Command=0x0000_0001 Status=0x7e
[1650608823029] [89418:1400921] CHIP: [TOO] Error: IM Error 0x0000057E: General error: 0x7e (UNSUPPORTED_ACCESS)

step5:

[1651237326.502624][18702:18707] CHIP:DMG: InvokeResponseMessage =
[1651237326.502644][18702:18707] CHIP:DMG: {
[1651237326.502665][18702:18707] CHIP:DMG:         suppressResponse = false, 
[1651237326.502684][18702:18707] CHIP:DMG:         InvokeResponseIBs =
[1651237326.502714][18702:18707] CHIP:DMG:         [
[1651237326.502730][18702:18707] CHIP:DMG:                 InvokeResponseIB =
[1651237326.502761][18702:18707] CHIP:DMG:                 {
[1651237326.502779][18702:18707] CHIP:DMG:                         CommandStatusIB =
[1651237326.502803][18702:18707] CHIP:DMG:                         {
[1651237326.502821][18702:18707] CHIP:DMG:                                 CommandPathIB =
[1651237326.502846][18702:18707] CHIP:DMG:                                 {
[1651237326.502868][18702:18707] CHIP:DMG:                                         EndpointId = 0x0,
[1651237326.502895][18702:18707] CHIP:DMG:                                         ClusterId = 0x30,
[1651237326.502918][18702:18707] CHIP:DMG:                                         CommandId = 0x4,
[1651237326.502940][18702:18707] CHIP:DMG:                                 },
[1651237326.502968][18702:18707] CHIP:DMG:                                 
[1651237326.502984][18702:18707] CHIP:DMG:                                 StatusIB =
[1651237326.503001][18702:18707] CHIP:DMG:                                 {
[1651237326.503019][18702:18707] CHIP:DMG:                                         status = 0x7e (UNSUPPORTED_ACCESS),
[1651237326.503050][18702:18707] CHIP:DMG:                                 },
[1651237326.503079][18702:18707] CHIP:DMG:                                 
[1651237326.503097][18702:18707] CHIP:DMG:                         },
[1651237326.503126][18702:18707] CHIP:DMG:                         
[1651237326.503144][18702:18707] CHIP:DMG:                 },
[1651237326.503172][18702:18707] CHIP:DMG:                 
[1651237326.503188][18702:18707] CHIP:DMG:         ],
[1651237326.503217][18702:18707] CHIP:DMG:         
[1651237326.503234][18702:18707] CHIP:DMG:         InteractionModelRevision = 1
[1651237326.503251][18702:18707] CHIP:DMG: },
[1651237326.503316][18702:18707] CHIP:DMG: Received Command Response Status for Endpoint=0 Cluster=0x0000_0030 Command=0x0000_0004 Status=0x7e
[1651237326.503345][18702:18707] CHIP:TOO: Error: IM Error 0x0000057E: General error: 0x7e (UNSUPPORTED_ACCESS)

step8:

[1651482593.056004][3607:3612] CHIP:DMG: ICR moving to [ResponseRe]
[1651482593.056073][3607:3612] CHIP:DMG: InvokeResponseMessage =
[1651482593.056111][3607:3612] CHIP:DMG: {
[1651482593.056148][3607:3612] CHIP:DMG:         suppressResponse = true, 
[1651482593.056185][3607:3612] CHIP:DMG:         InvokeResponseIBs =
[1651482593.056231][3607:3612] CHIP:DMG:         [
[1651482593.056267][3607:3612] CHIP:DMG:                 InvokeResponseIB =
[1651482593.056320][3607:3612] CHIP:DMG:                 {
[1651482593.056359][3607:3612] CHIP:DMG:                         CommandStatusIB =
[1651482593.056406][3607:3612] CHIP:DMG:                         {
[1651482593.056450][3607:3612] CHIP:DMG:                                 CommandPathIB =
[1651482593.056499][3607:3612] CHIP:DMG:                                 {
[1651482593.056550][3607:3612] CHIP:DMG:                                         EndpointId = 0x0,
[1651482593.056605][3607:3612] CHIP:DMG:                                         ClusterId = 0x3,
[1651482593.056657][3607:3612] CHIP:DMG:                                         CommandId = 0x1,
[1651482593.056707][3607:3612] CHIP:DMG:                                 },
[1651482593.056763][3607:3612] CHIP:DMG:                                 
[1651482593.056808][3607:3612] CHIP:DMG:                                 StatusIB =
[1651482593.056858][3607:3612] CHIP:DMG:                                 {
[1651482593.056907][3607:3612] CHIP:DMG:                                         status = 0x00 (SUCCESS),
[1651482593.056961][3607:3612] CHIP:DMG:                                 },
[1651482593.057011][3607:3612] CHIP:DMG:                                 
[1651482593.057055][3607:3612] CHIP:DMG:                         },
[1651482593.057107][3607:3612] CHIP:DMG:                         
[1651482593.057147][3607:3612] CHIP:DMG:                 },
[1651482593.057195][3607:3612] CHIP:DMG:                 
[1651482593.057231][3607:3612] CHIP:DMG:         ],
[1651482593.057275][3607:3612] CHIP:DMG:         
[1651482593.057311][3607:3612] CHIP:DMG:         InteractionModelRevision = 1
[1651482593.057346][3607:3612] CHIP:DMG: },
[1651482593.057432][3607:3612] CHIP:DMG: Received Command Response Status for Endpoint=0 Cluster=0x0000_0003 Command=0x0000_0001 Status=0x0
[1651482593.057492][3607:3612] CHIP:DMG: ICR moving to [AwaitingDe]

step9:

[1650440008.081437][6696:6701] CHIP:DMG: Received Command Response Data, Endpoint=0 Cluster=0x0000_0031 Command=0x0000_0001
[1650440008.081495][6696:6701] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_0031 Command 0x0000_0001
[1650440008.081615][6696:6701] CHIP:TOO:   ScanNetworksResponse: {
[1650440008.081668][6696:6701] CHIP:TOO:     networkingStatus: 0
[1650440008.081763][6696:6701] CHIP:TOO:     wiFiScanResults: 15 entries
[1650440008.081841][6696:6701] CHIP:TOO:       [1]: {
[1650440008.081865][6696:6701] CHIP:TOO:         Security: 12
[1650440008.081890][6696:6701] CHIP:TOO:         Ssid: 63686970736574757033
[1650440008.081913][6696:6701] CHIP:TOO:         Bssid: 0C0E764D4144
[1650440008.081935][6696:6701] CHIP:TOO:         Channel: 11
[1650440008.081957][6696:6701] CHIP:TOO:         WiFiBand: 0
[1650440008.081980][6696:6701] CHIP:TOO:         Rssi: -10
[1650440008.082004][6696:6701] CHIP:TOO:        }
[1650440008.082036][6696:6701] CHIP:TOO:       [2]: {
[1650440008.082058][6696:6701] CHIP:TOO:         Security: 12
[1650440008.082081][6696:6701] CHIP:TOO:         Ssid: 7A69676265652D7468726561642D3547487A
[1650440008.082103][6696:6701] CHIP:TOO:         Bssid: 6C198FC83ABE
[1650440008.082125][6696:6701] CHIP:TOO:         Channel: 44
[1650440008.082147][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.082168][6696:6701] CHIP:TOO:         Rssi: -32
[1650440008.082189][6696:6701] CHIP:TOO:        }
[1650440008.082220][6696:6701] CHIP:TOO:       [3]: {
[1650440008.082243][6696:6701] CHIP:TOO:         Security: 8
[1650440008.082266][6696:6701] CHIP:TOO:         Ssid: 47524C50726976617465
[1650440008.082288][6696:6701] CHIP:TOO:         Bssid: 6032B197B89F
[1650440008.082309][6696:6701] CHIP:TOO:         Channel: 153
[1650440008.082331][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.082352][6696:6701] CHIP:TOO:         Rssi: -37
[1650440008.082373][6696:6701] CHIP:TOO:        }
[1650440008.082404][6696:6701] CHIP:TOO:       [4]: {
[1650440008.082426][6696:6701] CHIP:TOO:         Security: 8
[1650440008.082448][6696:6701] CHIP:TOO:         Ssid:
[1650440008.082469][6696:6701] CHIP:TOO:         Bssid: 6632B197B89F
[1650440008.082491][6696:6701] CHIP:TOO:         Channel: 153
[1650440008.082512][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.082534][6696:6701] CHIP:TOO:         Rssi: -34
[1650440008.082555][6696:6701] CHIP:TOO:        }
[1650440008.082585][6696:6701] CHIP:TOO:       [5]: {
[1650440008.082607][6696:6701] CHIP:TOO:         Security: 8
[1650440008.082629][6696:6701] CHIP:TOO:         Ssid: 7A6967626565315F3547
[1650440008.082651][6696:6701] CHIP:TOO:         Bssid: E848B8C2123A
[1650440008.082689][6696:6701] CHIP:TOO:         Channel: 153
[1650440008.082710][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.082732][6696:6701] CHIP:TOO:         Rssi: -50
[1650440008.082753][6696:6701] CHIP:TOO:        }
[1650440008.082783][6696:6701] CHIP:TOO:       [6]: {
[1650440008.082806][6696:6701] CHIP:TOO:         Security: 8
[1650440008.082828][6696:6701] CHIP:TOO:         Ssid: 47524C50726976617465
[1650440008.082850][6696:6701] CHIP:TOO:         Bssid: C006C3F95EEB
[1650440008.082871][6696:6701] CHIP:TOO:         Channel: 153
[1650440008.082893][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.082914][6696:6701] CHIP:TOO:         Rssi: -45
[1650440008.082935][6696:6701] CHIP:TOO:        }
[1650440008.082965][6696:6701] CHIP:TOO:       [7]: {
[1650440008.082987][6696:6701] CHIP:TOO:         Security: 8
[1650440008.083009][6696:6701] CHIP:TOO:         Ssid: 47524C50726976617465
[1650440008.083031][6696:6701] CHIP:TOO:         Bssid: C006C3F95F31
[1650440008.083052][6696:6701] CHIP:TOO:         Channel: 48
[1650440008.083074][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.083095][6696:6701] CHIP:TOO:         Rssi: -51
[1650440008.083117][6696:6701] CHIP:TOO:        }
[1650440008.083146][6696:6701] CHIP:TOO:       [8]: {
[1650440008.083168][6696:6701] CHIP:TOO:         Security: 8
[1650440008.083192][6696:6701] CHIP:TOO:         Ssid: 4449524543542D564C4445534B544F502D3735334B364B4C6D735A55
[1650440008.083214][6696:6701] CHIP:TOO:         Bssid: E66A6A36F393
[1650440008.083236][6696:6701] CHIP:TOO:         Channel: 153
[1650440008.083257][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.083278][6696:6701] CHIP:TOO:         Rssi: -67
[1650440008.083299][6696:6701] CHIP:TOO:        }
[1650440008.083329][6696:6701] CHIP:TOO:       [9]: {
[1650440008.083352][6696:6701] CHIP:TOO:         Security: 8
[1650440008.083374][6696:6701] CHIP:TOO:         Ssid:
[1650440008.083395][6696:6701] CHIP:TOO:         Bssid: 1627F5374EC7
[1650440008.083417][6696:6701] CHIP:TOO:         Channel: 153
[1650440008.083438][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.083459][6696:6701] CHIP:TOO:         Rssi: -67
[1650440008.083480][6696:6701] CHIP:TOO:        }
[1650440008.083510][6696:6701] CHIP:TOO:       [10]: {
[1650440008.083533][6696:6701] CHIP:TOO:         Security: 8
[1650440008.083555][6696:6701] CHIP:TOO:         Ssid: 47524C50726976617465
[1650440008.083576][6696:6701] CHIP:TOO:         Bssid: 1027F5374EC7
[1650440008.083598][6696:6701] CHIP:TOO:         Channel: 153
[1650440008.083619][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.083640][6696:6701] CHIP:TOO:         Rssi: -70
[1650440008.083661][6696:6701] CHIP:TOO:        }
[1650440008.083691][6696:6701] CHIP:TOO:       [11]: {
[1650440008.083714][6696:6701] CHIP:TOO:         Security: 8
[1650440008.083735][6696:6701] CHIP:TOO:         Ssid: 47524C50726976617465
[1650440008.083758][6696:6701] CHIP:TOO:         Bssid: B0BE7653B01B
[1650440008.083779][6696:6701] CHIP:TOO:         Channel: 40
[1650440008.083800][6696:6701] CHIP:TOO:         WiFiBand: 2
[1650440008.083821][6696:6701] CHIP:TOO:         Rssi: -68
[1650440008.083843][6696:6701] CHIP:TOO:        }
[1650440008.083872][6696:6701] CHIP:TOO:       [12]: {
[1650440008.083895][6696:6701] CHIP:TOO:         Security: 8
[1650440008.083917][6696:6701] CHIP:TOO:         Ssid: 7A696762656531
[1650440008.083939][6696:6701] CHIP:TOO:         Bssid: E848B8C2123B
[1650440008.083960][6696:6701] CHIP:TOO:         Channel: 9
[1650440008.083981][6696:6701] CHIP:TOO:         WiFiBand: 0
[1650440008.084002][6696:6701] CHIP:TOO:         Rssi: -26
[1650440008.084024][6696:6701] CHIP:TOO:        }
[1650440008.084054][6696:6701] CHIP:TOO:       [13]: {
[1650440008.084077][6696:6701] CHIP:TOO:         Security: 12
[1650440008.084099][6696:6701] CHIP:TOO:         Ssid: 7A6967626565686F6D65
[1650440008.084121][6696:6701] CHIP:TOO:         Bssid: 0C0E764CB140
[1650440008.084142][6696:6701] CHIP:TOO:         Channel: 10
[1650440008.084164][6696:6701] CHIP:TOO:         WiFiBand: 0
[1650440008.084185][6696:6701] CHIP:TOO:         Rssi: -38
[1650440008.084206][6696:6701] CHIP:TOO:        }
[1650440008.084236][6696:6701] CHIP:TOO:       [14]: {
[1650440008.084259][6696:6701] CHIP:TOO:         Security: 12
[1650440008.084281][6696:6701] CHIP:TOO:         Ssid: 43686970736574757034
[1650440008.084303][6696:6701] CHIP:TOO:         Bssid: 0C0E764EF338
[1650440008.084324][6696:6701] CHIP:TOO:         Channel: 11
[1650440008.084345][6696:6701] CHIP:TOO:         WiFiBand: 0
[1650440008.084367][6696:6701] CHIP:TOO:         Rssi: -25
[1650440008.084388][6696:6701] CHIP:TOO:        }
[1650440008.084418][6696:6701] CHIP:TOO:       [15]: {
[1650440008.084441][6696:6701] CHIP:TOO:         Security: 8
[1650440008.084463][6696:6701] CHIP:TOO:         Ssid: 47524C50726976617465
[1650440008.084485][6696:6701] CHIP:TOO:         Bssid: 6032B197B89E
[1650440008.084507][6696:6701] CHIP:TOO:         Channel: 11
[1650440008.084528][6696:6701] CHIP:TOO:         WiFiBand: 0
[1650440008.084549][6696:6701] CHIP:TOO:         Rssi: -29
[1650440008.084570][6696:6701] CHIP:TOO:        }
[1650440008.084594][6696:6701] CHIP:TOO:    }

!!!ACTUAL BEHAVIOR:

step4:
./chip-tool onoff on 6789 1 --commissioner-nodeid  332211  
[1652365754.533149][140447:140447] CHIP:TOO: Run command failure: ../../examples/chip-tool/commands/common/CHIPCommand.cpp:392: CHIP Error 0x00000032: Timeout

step5:
./chip-tool generalcommissioning commissioning-complete  6789 0 --commissioner-fabricid 332211
[1652365961.392920][140545:140545] CHIP:TOO: Run command failure: ../../examples/chip-tool/commands/common/CHIPCommand.cpp:392: CHIP Error 0x00000032: Timeout

step8:
./chip-tool identify identify-query 6789 0 --suppressResponse 1
[1652366249.357255][140652:140657] CHIP:DMG:                            StatusIB =
[1652366249.357260][140652:140657] CHIP:DMG:                            {
[1652366249.357276][140652:140657] CHIP:DMG:                                    status = 0xc3 (UNSUPPORTED_CLUSTER),
[1652366249.357280][140652:140657] CHIP:DMG:                            },
[1652366249.357296][140652:140657] CHIP:DMG: 
[1652366249.357300][140652:140657] CHIP:DMG:                    },
[1652366249.357317][140652:140657] CHIP:DMG: 
[1652366249.357332][140652:140657] CHIP:DMG:            },
[1652366249.357339][140652:140657] CHIP:DMG: 
[1652366249.357353][140652:140657] CHIP:DMG:    ],
[1652366249.357359][140652:140657] CHIP:DMG: 
[1652366249.357375][140652:140657] CHIP:DMG:    InteractionModelRevision = 1
[1652366249.357378][140652:140657] CHIP:DMG: },
[1652366249.357405][140652:140657] CHIP:DMG: Received Command Response Status for Endpoint=0 Cluster=0x0000_0003 Command=0x0000_0001 Status=0xc3
[1652366249.357424][140652:140657] CHIP:TOO: Error: IM Error 0x000005C3: General error: 0xc3 (UNSUPPORTED_CLUSTER)
[1652366249.357433][140652:140657] CHIP:DMG: ICR moving to [AwaitingDe]
[1652366249.357470][140652:140657] CHIP:EM: Sending Standalone Ack for MessageCounter:5287221 on exchange 27825i

step9:
./chip-tool networkcommissioning scan-networks 6789 0
[1652366349.593135][140715:140720] CHIP:DMG:                            StatusIB =
[1652366349.593141][140715:140720] CHIP:DMG:                            {
[1652366349.593157][140715:140720] CHIP:DMG:                                    status = 0x85 (INVALID_COMMAND),
[1652366349.593161][140715:140720] CHIP:DMG:                            },
[1652366349.593204][140715:140720] CHIP:DMG: 
[1652366349.593208][140715:140720] CHIP:DMG:                    },
[1652366349.593225][140715:140720] CHIP:DMG: 
[1652366349.593240][140715:140720] CHIP:DMG:            },
[1652366349.593246][140715:140720] CHIP:DMG: 
[1652366349.593261][140715:140720] CHIP:DMG:    ],
[1652366349.593267][140715:140720] CHIP:DMG: 
[1652366349.593391][140715:140720] CHIP:DMG:    InteractionModelRevision = 1
[1652366349.593410][140715:140720] CHIP:DMG: },
[1652366349.593431][140715:140720] CHIP:DMG: Received Command Response Status for Endpoint=0 Cluster=0x0000_0031 Command=0x0000_0000 Status=0x85
[1652366349.593465][140715:140720] CHIP:TOO: Error: IM Error 0x00000585: General error: 0x85 (INVALID_COMMAND)

chip-tool output:
[tc-idm-1.2.md](https://github.com/project-chip/connectedhomeip/files/8680246/tc-idm-1.2.md)

**Comment by andy31415** [comment]
@nansenLV - this bug report is somewhat hard to follow because the logs are very long and the verification steps are not easily found (I do not know how to get steps from the code like TC-IDM-1.2). Please help improve the report by adding the following:

- what platform is this tested on? Chip-tool and DUT data
- what are the TC-IDM-1.2 steps ? Link to test plan is ok, a easy to read list of commands would be awesome
- Describe what the expected results are. `Not expected results` does not tell me what is expected nor what it actually gives. I see `INVALID_COMMAND` so assuming that is wrong, but actually have no idea what TC-IDM-1.2 is.

**Comment by bzbarsky-apple** [comment]
Test plan is at https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/interactiondatamodel.adoc#tc-idm-1-2-invoke-response-action-from-dut-to-th

The failures above in step 4 and step 5 are due to https://github.com/project-chip/connectedhomeip/issues/17221 : we end up doing CASE resumption but the node id has changed, so things break.

For step 8, there  is in fact no identify cluster on endpoint 0; it's on endpoint 1.  So the command being executed is wrong.

For step 9, as of TE9 the spec and impl required a (possibly null) SSID for a WiFi ScanNetworks, but the command is not providing one .  That has since been changed in the spec and impl.

Duplicate of #17221.
